libs/watt/src/lib/components/progress-tracker/src/watt-progress-tracker-step.component.scss

```diff
@@ -0,0 +1,83 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use "@energinet-datahub/watt/utils" as watt;
+
+.watt-progress-tracker-step-pending {
+  --watt-progress-tracker-step-text-color: var(--watt-color-neutral-grey-600);
+  --watt-progress-tracker-step-bridge-color: var(--watt-color-neutral-grey-600);
+  --watt-progress-tracker-step-fill-color: transparent;
+}
+
+.watt-progress-tracker-step-executing {
+  --watt-progress-tracker-step-text-color: var(--watt-color-neutral-grey-900);
+  --watt-progress-tracker-step-bridge-color: var(--watt-color-primary);
+  --watt-progress-tracker-step-fill-color: transparent;
+}
+
+.watt-progress-tracker-step-completed {
+  --watt-progress-tracker-step-text-color: var(--watt-color-neutral-grey-900);
+  --watt-progress-tracker-step-bridge-color: var(--watt-color-primary);
+  --watt-progress-tracker-step-fill-color: var(--watt-color-primary);
+}
+
+.watt-progress-tracker-step-failed {
+  --watt-progress-tracker-step-text-color: var(--watt-color-state-danger);
+  --watt-progress-tracker-step-bridge-color: var(--watt-color-state-danger);
+  --watt-progress-tracker-step-fill-color: var(--watt-color-state-danger);
+}
+
+watt-progress-tracker-step {
+  display: contents;
+}
+
+watt-progress-tracker-step:not(:first-child)::before {
+  content: "";
+  display: block;
+  width: 100%;
+  min-width: var(--watt-space-m);
+  height: 1px;
+  margin: 0 var(--watt-space-s);
+  background-color: var(--watt-progress-tracker-step-bridge-color);
+}
+
+.watt-progress-tracker-step-icon {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 24px; /* Magic UX number */
```

Comment:
I think --watt-space-ml is what you are looking for

Reply:
Thanks! 7985258087d637d9f659f85b40b5b4815b1e25b5